includes/class-wc-gateway-dintero-hp.php

```diff
@@ -1099,17 +1099,32 @@ private function capture( $order, $transaction = null ) {
 				return false;
 			}
 
-			if ($this->extract('status', $transaction) === 'CAPTURED') {
+			$transaction_status = $this->extract('status', $transaction);
+
+			if ($transaction_status === 'CAPTURED') {
 				$order->add_order_note('Payment captured via Dintero, already captured from before.');
 				$order->save_meta_data();
 				return false;
 			};
 
-			if ($this->extract('status', $transaction) !== 'AUTHORIZED') {
+			if ($transaction_status === 'REFUNDED' ||
+				$transaction_status === 'PARTIALLY_REFUNDED' ||
+				$transaction_status === 'PARTIALLY_CAPTURED_REFUNDED') {
```

Comment:
maybe remove this status, it does not mean the transaction has been captured

Reply:
done